Add deprecation notice (#37)
metal3-io/mariadb-image: README.md

# Metal3 MariaDB Container

DEPRECATION WARNING

The last release of Metal3-io mariadb-image will be done for the

> v1.11 release of cluster-api-provider-metal3.

Reviewer: ```suggestion
v1.11 release of cluster-api-provider-metal3 (CAPM3).
```

> From v1.12 of cluster-api-prorvider-metal3 and the related

Reviewer 2: ```suggestion
From v1.12 of cluster-api-provider-metal3 and the related
```

> From v1.12 of cluster-api-prorvider-metal3 and the related

Reviewer: ```suggestion
From v1.12 release of CAPM3 and the related
```

> baremetal-operator release will be tested only with the upstream

Reviewer: ```suggestion
baremetal-operator (BMO) release will be tested only with the upstream
```

mariadb-operator.

> This repo contains the files needed to build the MariaDB image optionally used

Reviewer: Move this line under description header.

This repo contains the files needed to build the MariaDB image optionally used
by Metal3.

> DEPRECATION WARNING

Reviewer 3: nit: prefix with `##` to make it more visible
